sphinx/builders/html/__init__.py

```diff
@@ -1176,14 +1188,16 @@ def validate_html_static_path(app: Sphinx, config: Config) -> None:
 
 def validate_html_logo(app: Sphinx, config: Config) -> None:
     """Check html_logo setting."""
-    if config.html_logo and not path.isfile(path.join(app.confdir, config.html_logo)):
+    if config.html_logo and not path.isfile(path.join(app.confdir, config.html_logo)) \
+            and not urlparse(config.html_logo).scheme:
```

Comment:
Could you use parenthesis instead of backslash to line-continuation?
```suggestion
    if (config.html_logo and not path.isfile(path.join(app.confdir, config.html_logo)) and
            not urlparse(config.html_logo).scheme):
```